plugin-opentelemetry/src/lib.rs

```diff
@@ -0,0 +1,70 @@
+mod output;
+
+use alumet::plugin::rust::{deserialize_config, serialize_config, AlumetPlugin};
+use output::OpenTelemetryOutput;
+use serde::{Deserialize, Serialize};
+
+pub struct OpenTelemetryPlugin {
+    config: Config,
+}
+
+impl AlumetPlugin for OpenTelemetryPlugin {
+    fn name() -> &'static str {
+        "plugin-opentelemetry"
```

Comment:
```suggestion
        "opentelemetry"
```
By convention, we don't put "plugin" in the name of the plugin (see other plugins). I should add that to the dev book.